### PR TITLE
Allow missing expert curves in discrimination calibration curve plot

### PR DESCRIPTION
**Pull request by hannahilea**
...as encountered in internal Beacon project.

### CODE REVIEW

**Review by ericphanson — APPROVED** [review]
thanks for figuring this out! I probably should've done this as part of https://github.com/beacon-biosignals/Lighthouse.jl/pull/94 too. Having the empty axis here seems reasonable and then layout will be consistent.

File: src/plotting.jl [hunk 1]
```diff
@@ -154,12 +158,19 @@ function plot_binary_discrimination_calibration_curves!(subfig::FigurePosition,
...
 
-    per_expert = get_theme(subfig, :BinaryDiscriminationCalibrationCurves, :PerExpert; solid_color=:darkgrey,
-                           color=nothing)
-    set_from_kw!(per_expert, :linewidth, kw, 2)
-    ax = series_plot!(subfig, per_expert_calibration_curves, nothing; legend=nothing,
-                      title="Detection calibration", xlabel="Expert agreement rate",
-                      ylabel="Predicted positive probability", scatter=scatter_theme, per_expert...)
+    if ismissing(per_expert_calibration_curves)
+        ax = Axis(subfig; title="Detection calibration", xlabel="Expert agreement rate",
+                  ylabel="Predicted positive probability", legend=nothing)
```
Comment (ericphanson) [inline src/plotting.jl:163]: CI doesn't like the `legend` argument here, apparently:
```suggestion
        ax = Axis(subfig; title="Detection calibration", xlabel="Expert agreement rate",
                  ylabel="Predicted positive probability")
```

Comment (SimonDanisch) [reply src/plotting.jl:163]: Yeah that's Plots.jl speak ;)

**Comment by hannahilea** [comment]
thanks for fixing the legend failure!

**Comment by hannahilea** [comment]
...I added a couple more plots to percy, but no other changes; realized we weren't plotting the discrimination plots in isolation yet, like we do the others.

**Pull request merged**